src/main/java/com/danielagapov/spawn/Services/S3/IS3Service.java

```diff
@@ -0,0 +1,14 @@
+package com.danielagapov.spawn.Services.S3;
+
+
+import com.danielagapov.spawn.DTOs.UserDTO;
+
+import java.util.UUID;
+
+public interface IS3Service {
+    String putObject(byte[] file, String key);
+    void deleteObjectFromUser(UUID id);
+    String putObject(byte[] file);
+    UserDTO putObjectWithUser(byte[] file, UserDTO user);
+    UserDTO updateProfilePicture(byte[] file, UUID id);
+}
```

Comment:
```suggestion
    String putObjectWithKey(byte[] file, String key);
    void deleteObjectByUserId(UUID userId);
    String putObject(byte[] file);
    UserDTO putObjectWithUser(byte[] file, UserDTO user);
    UserDTO updateProfilePicture(byte[] file, UUID userId);
}
```

Specifying which kind of id + keeping consistent with your convention of "With_" in method names, like you did in `putObjectWithUser()`. These are suggestions for naming changes; whatever you end up going with, please apply to the methods in `S3Service.java`

Reply:
Renamed methods accordingly